src/Types/post.ts

```diff
@@ -0,0 +1,9 @@
+import { User } from './user';
+
+export interface Post {
+  userId: number,
+  id: number,
+  title: string,
+  body: string,
+  user: User | null
```

Comment:
Since each post should have a user and comments associated with it, consider adding a 'comments' property of type 'Comment[]' to the 'Post' interface. Additionally, import 'Comment' from './comment'.

Reply:
isn't fixed